src/test/java/uk/gov/hmcts/darts/hearings/mapper/GetAnnotationsResponseMapperTest.java

```diff
@@ -0,0 +1,96 @@
+package uk.gov.hmcts.darts.hearings.mapper;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.darts.common.entity.AnnotationDocumentEntity;
+import uk.gov.hmcts.darts.common.entity.AnnotationEntity;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+import uk.gov.hmcts.darts.common.util.CommonTestDataUtil;
+import uk.gov.hmcts.darts.hearings.model.Annotation;
+
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class GetAnnotationsResponseMapperTest {
+
+    @Test
+    void convertOk() {
```

Comment:
This one might benefit from adding `Given / When / Then` comments to delineate test setup from invocation and assertion, per https://hmcts-reform.slack.com/archives/C05HQ77QW4D/p1705503485484679. See what you think.

Reply:
done